File: app/src/main/java/fr/erban/dixitcompanion/game/turn/Turn.java
package fr.erban.dixitcompanion.game.turn;

import java.io.Serializable;

import fr.erban.dixitcompanion.game.player.Player;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Turn implements Serializable {

    private boolean noOneFound;

    private boolean everybodyFound;

    private Player storyTeller;
}

File: dixit-companion/src/main/java/fr/erban/dixitcompanion/game/turn/Turn.kt
package fr.erban.dixitcompanion.game.turn

import android.os.Parcelable
import fr.erban.dixitcompanion.game.player.PlayerBean
import fr.erban.dixitcompanion.game.turn.bean.VoteBean
import kotlinx.parcelize.Parcelize

@Parcelize
data class Turn(
    val storyTeller: PlayerBean? = null,
    val votes: List<VoteBean> = emptyList(),
    val whoFound: List<PlayerBean> = emptyList(),
    val noOneFound: Boolean = false,
    val everybodyFound: Boolean = false
) : Parcelable
